backend/apps/ecommerce/mail.py

```diff
@@ -27,30 +16,30 @@ def send_order_confirmation_mail(order: Order) -> None:
     """
 
     assert order.payment_status == Order.PaymentStatus.CAPTURED
-    user = order.user
-
-    content: EmailInput = {
-        "first_name": user.first_name,
-        "last_name": user.last_name,
-        "order_id": order.id,
-        "timestamp": order.timestamp.strftime("%d.%m.%Y, %H:%M:%S"),
-        "product": order.product.name,
-        "quantity": order.quantity,
-        "price": order.total_price,
+    user: User = order.user
+
+    receipt_details: list[TemplateVariables] = [
+        {"description": order.product.description, "amount": f"{order.product.price} NOK"}
+        for _ in range(order.quantity)
+    ]
+    template_variables: TemplateVariables = {
+        user.email: {
+            "product_url": f"{settings.BASE_URL}/ecommerce/fallback?orderId={order.id}",
+            "product_name": order.product.name,
+            "name": user.first_name,
+            "receipt_id": str(order.id),
+            "date": order.timestamp.strftime("%d.%m.%Y, %X"),
+            "receipt_details": receipt_details,
+            "total": f"{order.total_price} NOK",
+            "support_mail": "kontakt@rubberdok.no",
```

Comment:
Should this be hardcoded in or be a variable someplace?

Reply:
Added to env and moved to a `CONTACT_EMAIL` variable